drivers/syslog/Kconfig

```diff
@@ -102,15 +102,59 @@ config SYSLOG_TIMESTAMP
 	---help---
 		Prepend timestamp to syslog message.
 
+choice
+	prompt "Timestamp format"
+	depends on SYSLOG_TIMESTAMP
+
+config SYSLOG_TIMESTAMP_SYSTIME
+	bool "System time"
+	default y
+	---help---
+		Use the system timer for timestamp.
+
 config SYSLOG_TIMESTAMP_REALTIME
-	bool "Use wall-clock for syslog timestamp"
+	bool "Wall-clock time"
 	default n
-	depends on SYSLOG_TIMESTAMP
 	---help---
 		Use wall-clock (CLOCK_REALTIME) for timestamp.  By default,
 		CLOCK_MONOTONIC, if enabled, will be used or the system timer
 		is not.
 
+config SYSLOG_TIMESTAMP_FORMATED
```

Comment:
Why SYSLOG_TIMESTAMP_FORMATED can't work with CLOCK_MONOTONIC(use choice here)? From the code, the format work with all timing source.

Reply:
I am not sure what changes do you propose on the Kconfig options.

Can you please explain?

Reply:
I mean that:
1. Remove SYSLOG_TIMESTAMP_SYSTIME
2. Move SYSLOG_TIMESTAMP_FORMATED out of choice
Basically, keep SYSLOG_TIMESTAMP and SYSLOG_TIMESTAMP_REALTIME as before, and add a new option SYSLOG_TIMESTAMP_FORMATED.

Reply:
Ok, I reverted it back to the previous structure.
Please check now.